### PR TITLE
Schema updates for issue #861

### PR DESCRIPTION
**Pull request by stscirij**
pathloss and mask schema updates

### CODE REVIEW

File: jwst/datamodels/schemas/mask.schema.yaml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 allOf:
 - $ref: referencefile.schema.yaml
+- $ref: subarray.schema.yaml
+- $ref: keyword_preadpatt.schema.yaml
```
Comment (hbushouse) [inline jwst/datamodels/schemas/mask.schema.yaml]: MASK files use DETECTOR, SUBARRAY, and READPATT (in an or-like fashion) as CRDS selectors, so this looks good. I'm guessing the dq_init step code does not use any other keywords from the reference file.

File: jwst/datamodels/schemas/pathloss.schema.yaml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 title: Pathloss correction data model
 allOf:
 - $ref: referencefile.schema.yaml
+- $ref: keyword_pexptype.schema.yaml
```
Comment (hbushouse) [inline jwst/datamodels/schemas/pathloss.schema.yaml]: PATHLOSS ref files use only EXP_TYPE, in an or-like fashion, as a CRDS selector, so this looks good. Are any other keywords from the ref file used in the pathloss step code?

Comment (stscirij) [inline jwst/datamodels/schemas/pathloss.schema.yaml]: No

File: jwst/datamodels/schemas/mask.schema.yaml (hunk 1 above)
Comment (hbushouse) [inline jwst/datamodels/schemas/mask.schema.yaml]: 👍

File: jwst/datamodels/schemas/pathloss.schema.yaml (hunk 2 above)
Comment (hbushouse) [inline jwst/datamodels/schemas/pathloss.schema.yaml]: 👍

**Pull request merged**